test/integration/scheduler/scheduler_test.go

```diff
@@ -72,6 +72,41 @@ func PriorityTwo(pod *v1.Pod, nodeNameToInfo map[string]*schedulercache.NodeInfo
 	return []schedulerapi.HostPriority{}, nil
 }
 
+func TestSchedulerCreation(t *testing.T) {
```

Comment:
Seems like this could just be a unit test and use a fake clientset (`k8s.io/client-go/kubernetes/fake`)

Reply:
Done.  I have changed to use a fake client.  Ready to review ~~ thank you